src/main/java/org/opensearch/security/privileges/SecurityIndexAccessEvaluator.java

```diff
@@ -71,11 +75,13 @@ public SecurityIndexAccessEvaluator(final Settings settings, AuditLog auditLog,
         this.systemIndexMatcher = WildcardMatcher.from(
             settings.getAsList(ConfigConstants.SECURITY_SYSTEM_INDICES_KEY, ConfigConstants.SECURITY_SYSTEM_INDICES_DEFAULT)
         );
+        this.denylistIndexMatcher = WildcardMatcher.from(
+            settings.getAsList(ConfigConstants.SECURITY_INDICES_DENYLIST_KEY, ConfigConstants.SECURITY_INDICES_DENYLIST_KEY_DEFAULT)
+        );
```

Comment:
RE: The deny list should not be configurable
We can keep the deny list index matcher, but populate it with only the value for the security index.

Reply:
Just to be sure, we are updating the requirements  described on issue https://github.com/opensearch-project/security/issues/3038